### ISSUE TITLE
Error when clicking a navigationBar path link

### ISSUE THREAD

**Issue by wanderg84** [josdejong/jsoneditor#595]
https://github.com/josdejong/jsoneditor/commit/181140f733baddd2e643e18cf6f1fe1d516acb27#diff-9fd481e566a0f24de9121c0320b1503bL219

- Missing null check in above commit
- An error occurs if there is no 'frame' param.

console log :
ContextMenu.js:209 Uncaught TypeError: Cannot read property 'getBoundingClientRect' of undefined
    at ContextMenu.webpackJsonp../node_modules/jsoneditor/src/js/ContextMenu.js.ContextMenu.show (ContextMenu.js:209)
    at HTMLSpanElement.sepEl.onclick (TreePath.js:62)

**Comment by josdejong** [comment]
Thanks for reporting @wander84, that's a bug.

@meirotstein do you have an idea where this can originate from? It did work in the past.

**Comment by meirotstein** [comment]
@josdejong I will have a look

**Comment by meirotstein** [comment]
it is still working...
@wanderg84 can u please provide steps for reproduce? 
thanks!

**Comment by josdejong** [comment]
@meirotstein it happens when you click the triangle between two names:

![image](https://user-images.githubusercontent.com/568626/47455018-082b6b00-d7d1-11e8-891f-0cc298722f98.png)

This should open a dropdown with all properties in the object, but it gives this error in the console:

```
Uncaught TypeError: Cannot read property 'getBoundingClientRect' of undefined
    at ContextMenu.show (jsoneditor.js:32922)
    at HTMLSpanElement.sepEl.onclick (jsoneditor.js:35442)
```

**Comment by josdejong** [comment]
Ah, I think I caused this error after refactoring the ContextMenu some time ago. ContextMenu.show now requires a second argument in TreePath.js on line 62: `menu.show(sepEl);`. I will look into it soon (until you beat me Meir ;) )

**Comment by meirotstein** [comment]
@josdejong cool! nice fix!

**Comment by josdejong** [comment]
yeah, well, I should have tested better when changing the ContextMenu. It was changed from being relatively positioned to being absolutely positioned.

**Comment by josdejong** [comment]
Should be fixed now in `v5.25.0`

**Comment by wanderg84** [comment]
Thank you for your quick fix!
